airbyte-cdk/python/unit_tests/sources/file_based/scenarios/csv_scenarios.py

```diff
@@ -34,6 +35,158 @@
         }
     )
     .set_file_type("csv")
+    .set_expected_spec(
+        {
+            "documentationUrl": "https://docs.airbyte.com/integrations/sources/in_memory_files",
+            "connectionSpecification": {
+                "title": "InMemorySpec",
+                "description": "Used during spec; allows the developer to configure the cloud provider specific options\nthat are needed "
+                               "when users configure a file-based source.",
+                "type": "object",
+                "properties": {
+                    "streams": {
+                        "title": "The list of streams to sync",
+                        "description": "Streams defines the behavior for grouping files together that will be synced to the downstream "
+                                       "destination. Each stream has it own independent configuration to handle which files to sync, "
+                                       "how files should be parsed, and the validation of records against the schema.",
+                        "order": 10,
+                        "type": "array",
+                        "items": {
+                            "title": "FileBasedStreamConfig",
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "title": "Name",
+                                    "type": "string"
+                                },
+                                "file_type": {
```

Comment:
the file_type is part of the stream config. Does this mean the source can support multiple file types?

Reply:
It makes sense to me that each stream could correspond to a different file type.

Reply:
correct that would be the intention. Although it is still a single stream can only extract from one file type (although long term we can hopefully set us up to support multiple file types per string)